src/kic/api/viewsets.py

```diff
@@ -420,3 +423,71 @@ def get_queryset(self):
         if marked_vios:
             return qs.exclude(uuid__in=marked_vios)
         return qs
+
+
+class VerzoekProductViewSet(
+    CheckQueryParamsMixin,
+    mixins.CreateModelMixin,
+    mixins.DestroyModelMixin,
+    viewsets.ReadOnlyModelViewSet,
+):
+    """
+    Opvragen en bewerken van VERZOEK-PRODUCT relaties.
+
+    create:
+    Maak een VERZOEK-PRODUCT relatie aan.
+
+    Registreer een PRODUCT bij een VERZOEK. Er worden twee types van
+    relaties met andere objecten gerealiseerd:
+
+    **Er wordt gevalideerd op**
+    - geldigheid `verzoek` URL
+    - geldigheid `product` URL
+
+    list:
+    Alle VERZOEK-PRODUCT relaties opvragen.
+
+    Deze lijst kan gefilterd wordt met query-string parameters.
+
+    retrieve:
+    Een specifieke VERZOEK-PRODUCT relatie opvragen.
+
+    Een specifieke VERZOEK-PRODUCT relatie opvragen.
+
+    update:
+    Werk een VERZOEK-PRODUCT relatie in zijn geheel bij.
+
+    Je mag enkel de gegevens van de relatie bewerken, en niet de relatie zelf
+    aanpassen.
+
+    **Er wordt gevalideerd op**
+    - `product` URL en `verzoek` URL mogen niet veranderen
```

Comment:
Add that you're not allowed to change product identificatie either?

Reply:
Instead, I removed the update/patch docs since we don't support those operations.